go/app/main.go

```diff
@@ -25,18 +40,74 @@ func root(c echo.Context) error {
 	return c.JSON(http.StatusOK, res)
 }
 
+func dataJson() Items{
+	//Open our jsonFile
+	jsonFile, err := os.Open("items.json")
+	if err != nil {
+		fmt.Println(err)
```

Comment:
If you do not want to handle the error, you should pass it to the caller so that it can be handled by the caller. In Go these functions usually have return types of (valueType, error).